backend/src/graphql/resolver/CommunityResolver.ts

```diff
@@ -1,58 +1,21 @@
 import { Resolver, Query, Authorized } from 'type-graphql'
 
 import { Community } from '@model/Community'
+import { Community as DbCommunity } from '@entity/Community'
 
 import { RIGHTS } from '@/auth/RIGHTS'
-import CONFIG from '@/config'
 
 @Resolver()
 export class CommunityResolver {
-  @Authorized([RIGHTS.GET_COMMUNITY_INFO])
-  @Query(() => Community)
-  getCommunityInfo(): Community {
-    return new Community({
-      name: CONFIG.COMMUNITY_NAME,
-      description: CONFIG.COMMUNITY_DESCRIPTION,
-      url: CONFIG.COMMUNITY_URL,
-      registerUrl: CONFIG.COMMUNITY_REGISTER_URL,
```

Comment:
Is this still in the config?

Reply:
Yes, some of these config properties are still in use and with the next steps of federation they will be used more than now